FibonacciNumbers.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class FibonacciNumbers {
+  public static void main(String[] args) {
+    int max = 2147483647, num1 = 0, num2 = 1, sum = 0, a = 0;
+    boolean ind1 = false, ind2 = false;
+    Scanner in = new Scanner (System.in);
+    try {
+      while (ind1 == false) {
+        System.out.println("Enter a number");
+        a = in.nextInt();
+        if (a < 0) {
+          System.out.println("Your number is less than zero. Repeat, please");
+        } else {
+          ind1 = true;
+        }
+      }
+    } catch (Exception e) {
+        System.out.println("You did not enter a number or your number is very large. The program stops working");
+        return;       
+    }
+    while (sum < max * 0.8) {

```

Comment:
Because the maximal value for type Integer is 2147483647, the maximal Fibonacci number for this type is 1836311903 and it is about 85% of the MAX_INTEGER. I used the number, which little less than 1836311903, for receiving the last Fibonacci number, that cycle create one more iteration. And this number it is 80% of the MAX_INTEGER